src/preload-hover.js

```diff
@@ -19,17 +19,23 @@ export default class PreloadHover {
 
     domScopes.forEach(domScope => {
       const links = [...domScope.getElementsByTagName('a')];
-
+      let timer;
       let uniqueLinks = uniqBy(links, 'href');
 
       uniqueLinks.forEach(link => {
         link.addEventListener('mouseover', () => {
-          const preload = document.createElement('link');
-
-          preload.setAttribute('rel', 'preload');
-          preload.setAttribute('href', link.href);
+          timer = setTimeout(() =>{
+            const preload = document.createElement('link');
+            
+            preload.setAttribute('rel', 'preload');
+            preload.setAttribute('href', link.href);
+
+            head.appendChild(preload);
+            }, 200);
```

Comment:
Is 200ms too long to wait? Maybe 50ms would be more appropriate? Opinions? http://instantclick.io/click-test